learning-development/accessibility.qmd

```diff
@@ -4,33 +4,67 @@ title: "Accessibility"
+- [Colour contrast matrix (comparing multiple colours](https://jansensan.github.io/color-contrast-matrix/)
+
+Colour blindness:
+
+- [Adobe colour accessibility (gives a pass / fail)](https://color.adobe.com/create/color-accessibility)
+- [Coblis (simulates colour blindness for uploaded images)](https://www.color-blindness.com/coblis-color-blindness-simulator/)
+- [ColourBlindCheck (R package for simulating colour blindness for a colour palette)](https://github.com/Nowosad/colorblindcheck)
+
+Grey scale:
+
+- You can often check colours in grey scale by using a print preview and switching the colour mode
+- [ColToGrey function within the DescTools R package](https://andrisignorell.github.io/DescTools/reference/ColToGrey.html)
+
+For more advice about colour in charts and visualisations specifically, including more tools and resources for checking colours yourself, see the [Analysis function colours in charts guidance](https://analysisfunction.civilservice.gov.uk/policy-store/data-visualisation-colours-in-charts/#section-9).
+
+---
+
+## Free tools for reviewing web pages
+
+- [axe DevTools](https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?pli=1), free chrome extension and paid versions, we recommend this (and GDS tend to recommend too)
+
+- [Google Lighthouse](https://developer.chrome.com/docs/lighthouse/overview), built into Google Chrome browser, and will catch some basic accessibility things amongst other web related issues
+
+'Bookmarklets' are bookmarks that instead of saving a URL to a page, save a piece of Javascript code that executes on the page you are looking at. There's some nice [accessibility focused bookmarklets](https://design.education.gov.uk/accessibility/assurance) that highlight specific types of mark up such as headers and lists so you can easily check if it matches what you'd expect while on any webpage. Very nifty and low effort too!
+
 ---
 
-Accessibility tools you can use
+## Assistive technology
 
-If wanting to check anything around colour:
-	- Contrast - webaim / https://jansensan.github.io/color-contrast-matrix/
-	- Colour blindness -
-	- Colour blindness simulation - 
-	- Greyscale - 
+No automated tools fully cover accessibility, manual testing is always required, so if you want to do manual testing have a look at testing out the assistive software commonly used by users yourself.
 
-Helpful browser addins
+Magnifier:
 
-These 'bookmarklets' are bookmarks that instead of saving a URL to a page, save a piece of Javascript code that executes on the page you are looking at, usually to highlight specific mark up. Very nifty!
+- Built into Windows ([guidance for how to turn on Magnifier](https://support.microsoft.com/en-gb/windows/use-magnifier-to-make-things-on-the-screen-easier-to-see-414948ba-8b1c-d3bd-8615-0e5e32204198#:~:text=To%20quickly%20turn%20on%20Magnifier,turn%20on%20the%20Magnifier%20switch.))
```

Comment:
the way this is phrased makes it a bit confusing as there are two separate things under the Magnifier heading and one of them is actually called Magnifier and one isn't

Reply:
Made this change in my PR comments commit